### ISSUE TITLE
Does not build on `nixos-21.05`

### ISSUE THREAD

**Issue by cprussin** [Synthetica9/nix-linter#52]
The `nixos-21.05` release just got cut the other day and `nix-linter` does not build with it:

```
CallStack (from HasCallStack):
  $, called at libraries/Cabal/Cabal/Distribution/Simple/Configure.hs:1024:20 in Cabal-3.2.1.0:Distribution.Simple.Configure
  configureFinalizedPackage, called at libraries/Cabal/Cabal/Distribution/Simple/Configure.hs:477:12 in Cabal-3.2.1.0:Distribution.Simple.Configure
  configure, called at libraries/Cabal/Cabal/Distribution/Simple.hs:625:20 in Cabal-3.2.1.0:Distribution.Simple
  confHook, called at libraries/Cabal/Cabal/Distribution/Simple/UserHooks.hs:65:5 in Cabal-3.2.1.0:Distribution.Simple.UserHooks
  configureAction, called at libraries/Cabal/Cabal/Distribution/Simple.hs:180:19 in Cabal-3.2.1.0:Distribution.Simple
  defaultMainHelper, called at libraries/Cabal/Cabal/Distribution/Simple.hs:116:27 in Cabal-3.2.1.0:Distribution.Simple
  defaultMain, called at Setup.hs:2:8 in main:Main
Setup: Encountered missing or private dependencies:
hnix >=0.8 && <0.11
```

Looks like there are already a PR open to fix this (#51), any chance of getting that merged & released (perhaps in a separate branch)?

**Comment by expipiplus1** [comment]
Closed by #53 and https://github.com/NixOS/nixpkgs/projects

**Comment by cprussin** [comment]
Thank you!
